gatsby-remark-oembed/.eslintrc

```diff
@@ -1,13 +1,12 @@
 {
   "env": {
+    "es6": true,
```

Comment:
This is what allows for the linter to recognize Promises as part of the language. The ESLint configuration still needs a bit of work, but that will be in a subsequent PR.